elasticsearch.md

```diff
@@ -700,6 +700,19 @@ For example, `test` is the username and `$6$rounds=65535$d07dnv4N$QeErsDT9Mz.ZoE
 
 [Impersonation](details/impersonation.md) is supported by this rule without an extra configuration.
 
+#### `token_authentication`
+
+```yaml
+token_authentication:
+   token: "Bearer AAEAAWVsYXN0aWMva2liYW5hL3Rva2Vu"
+   username: admin                # username used after successful authentication
+   header: x-custom-authorization # optional, defaults to 'Authorization`
+```
+
+The authentication rule that accepts tokens sent in the HTTP header (`Authorization` header by default). Clients will need to provide the header e.g. `Authorization: Bearer AAEAAWVsYXN0aWMva2liYW5hL3Rva2Vu` where "Bearer AAEAAWVsYXN0aWMva2liYW5hL3Rva2Vu" is token value set in the `token` field.
```

Comment:
* tokens -> token 
* the `Authorization` header is the default if no custom header name is configured in the `token_authentication.header` field.
* that last sentence ("Clients will need...") are too technical/artificial. Let's make it more natural.